app/v2/controllers/AmendForeignPropertyPeriodSummaryController.scala

```diff
@@ -96,11 +96,24 @@ class AmendForeignPropertyPeriodSummaryController @Inject()(val authService: Enr
 
   private def errorResult(errorWrapper: ErrorWrapper) =
     errorWrapper.error match {
-      case BadRequestError | NinoFormatError | TaxYearFormatError | BusinessIdFormatError | SubmissionIdFormatError | RuleTaxYearRangeInvalidError |
-          RuleTaxYearNotSupportedError | RuleIncorrectOrEmptyBodyError | RuleTypeOfBusinessIncorrectError | MtdErrorWithCode(
-            CountryCodeFormatError.code) | MtdErrorWithCode(RuleCountryCodeError.code) | MtdErrorWithCode(ValueFormatError.code) | MtdErrorWithCode(
-            RuleBothExpensesSuppliedError.code) | MtdErrorWithCode(RuleIncorrectOrEmptyBodyError.code) | MtdErrorWithCode(
-            RuleDuplicateCountryCodeError.code) =>
+      case _
```

Comment:
Can we refactor line 65 with the Hateoas links to be like how it is done [here](https://github.com/hmrc/self-employment-business-api/blob/main/app/v1/controllers/RetrieveAnnualSubmissionController.scala)